lib/vector-buffers/src/test/common/helpers.rs

```diff
@@ -0,0 +1,102 @@
+use std::{future::Future, path::Path, str::FromStr};
+
+use once_cell::sync::Lazy;
+use temp_dir::TempDir;
+use tracing_fluent_assertions::{AssertionRegistry, AssertionsLayer};
+use tracing_subscriber::{filter::LevelFilter, layer::SubscriberExt, Layer, Registry};
+
+/*
+    Helper code for getting tracing data from a test:
+
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::TRACE)
+        .with_span_events(tracing_subscriber::fmt::format::FmtSpan::FULL)
+        .with_test_writer()
+        .init();
+*/
```

Comment:
Want to keep this? Looks like it's in the helper function below.

Reply:
Good call. 👍🏻